variables.tf

```diff
@@ -0,0 +1,110 @@
+variable "cluster_arn" {
+  type        = string
+  description = "The ARN of the ECS cluster"
+  default     = null
+}
+
+variable "config" {
+  description = "A map containing all labeler configuration options"
+  type = object({
+    log-level                   = optional(string)
+    region                      = optional(string)
+    organizations-zone-name     = optional(string)
+    audit-zone-name             = optional(string)
+    single-account-id           = optional(string)
+    frameworks                  = optional(list(string), [])
+    allowed-account-ids         = optional(list(string), [])
+    denied-account-ids          = optional(list(string), [])
+    allowed-regions             = optional(list(string), [])
+    denied-regions              = optional(list(string), [])
+    export-path                 = optional(string)
+    export-metrics-only         = optional(bool, false)
+    to-json                     = optional(bool, false)
+    report-closed-findings-days = optional(number)
+    report-suppressed-findings  = optional(bool, false)
+    account-thresholds          = optional(string)
+    zone-thresholds             = optional(string)
+    security-hub-query-filter   = optional(string)
+    validate-metadata-file      = optional(string)
+  })
+  default = {}
+  validation {
+    condition     = length(compact([var.config.single-account-id, var.config.audit-zone-name, var.config.organizations-zone-name])) == 1
+    error_message = "Parameters organizations-zone-name, audit-zone-name and single-account-id are mutually exclusive"
```

Comment:
what is audit-zone-name?

Reply:
/shrug ... it's passed to the cli

Reply:
https://awsenergylabelerlib.readthedocs.io/en/latest/usage.html ?

Reply:
https://aws-energy-labeler-cli.readthedocs.io/en/latest/usage.html